src/ProjectNeon/Assets/Scripts/CardsSelectController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using UnityEngine;
+
+public class CardsSelectController : MonoBehaviour
+{
+    [SerializeField] ConfirmCardSelection ConfirmCardSelection;
+    [SerializeField] CardPlayZones playZones;
+    [SerializeField] GameObject targetPoint;
+    [SerializeField] GameObject handArea;
+    [SerializeField] float cardDelta;
+
+    CardPresenter[] playerCards;
+    private void Start()
+    {
+        ConfirmCardSelection.onConfirmation.Subscribe(FindSelectedCards, this);
+    }
+    public void FindSelectedCards()
+    {
+        playerCards = transform.GetComponentsInChildren<CardPresenter>();
+        StartCoroutine(MoveToBottom());
+    }
+    // @todo #133: 40min Change card rotation while moving down
+    // @todo #133: 20min Change card scale while moving down
+    IEnumerator MoveToBottom()
+    {
+        handArea.SetActive(false);
+        Vector3 targetPosition;
+        int mediane = playerCards.Length / 2;
+        bool stop = false;
+        while (!stop)
+        {
+            yield return null;
+            stop = true;
+            for (int i = 0; i < playerCards.Length; i++)
```

Comment:
Why are we moving each individual card, instead of moving the entire Hand Panel?

Reply:
I had several reasons for this. Mainly because this way I can set a specific position (in the long term scale and rotation) for each card separately. If you assumed work with the panel, but not with separate cards - it is possible to remake this feature. However, I think that the implemented option is more universal and convenient.